README.md

```diff
@@ -1,51 +1,34 @@
-# JSOSuite.jl
-
-[![docs-stable][docs-stable-img]][docs-stable-url] [![docs-dev][docs-dev-img]][docs-dev-url] [![build-ci][build-ci-img]][build-ci-url] [![codecov][codecov-img]][codecov-url] [![release][release-img]][release-url]
-
-[docs-stable-img]: https://img.shields.io/badge/docs-stable-blue.svg
-[docs-stable-url]: https://JuliaSmoothOptimizers.github.io/JSOSuite.jl/stable
-[docs-dev-img]: https://img.shields.io/badge/docs-dev-purple.svg
-[docs-dev-url]: https://JuliaSmoothOptimizers.github.io/JSOSuite.jl/dev
-[build-ci-img]: https://github.com/JuliaSmoothOptimizers/JSOSuite.jl/workflows/CI/badge.svg?branch=main
-[build-ci-url]: https://github.com/JuliaSmoothOptimizers/JSOSuite.jl/actions
-[codecov-img]: https://codecov.io/gh/JuliaSmoothOptimizers/JSOSuite.jl/branch/main/graph/badge.svg
-[codecov-url]: https://codecov.io/gh/JuliaSmoothOptimizers/JSOSuite.jl
-[release-img]: https://img.shields.io/github/v/release/JuliaSmoothOptimizers/JSOSuite.jl.svg?style=flat-square
-[release-url]: https://github.com/JuliaSmoothOptimizers/JSOSuite.jl/releases
-
-One stop solutions for all things optimization.
+# JSOSuite
```

Comment:
It is probably worth adding somewhere adding that we use the COPIERTemplate in case somebody tries to update CI and stuff. what do you think?